packages/website/src/helpers/git.ts

```diff
@@ -29,7 +29,7 @@ export async function init(repo: string, ref: string) {
   } else {
     await fetch({ ...baseOpts, singleBranch: true, depth: 1 });
     await checkout({ fs, dir, ref });
-    await fetch({ ...baseOpts, singleBranch: true, depth: 1 });
+    await pull({ ...baseOpts, singleBranch: true });
```

Comment:
you sure this is going to work? have you been able to test? I guess I am just skeptical for variety reasons